src/css/_utilities.scss

```diff
@@ -94,11 +96,15 @@
   order: $value;
 }
 
-%icon-default {
+%overlay-space {
```

Comment:
'overlay-space' feels a little vague to me. Would 'fill-parent' be an accurate description?